Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Base image for GenAIComps based OPEA Python applications
+# Build: docker build -t opea/comps-base -f Dockerfile .
+
+ARG IMAGE_NAME=python
+ARG IMAGE_TAG=3.11-slim
+
+FROM ${IMAGE_NAME}:${IMAGE_TAG} AS base
+
+ENV HOME=/home/user
+
+RUN useradd -m -s /bin/bash user && \
+    mkdir -p $HOME && \
+    chown -R user $HOME
+
+# get security updates
+RUN apt-get update && apt-get upgrade -y && \
```

Comment:
Once the `upgrade` is done, some old packages may become stale and no longer needed and it that case:
```
apt-get autoclean && \
apt-get autoremove -y
```
would cleanup even furthur.

Reply:
I don't think that is necessary in this particular case.  `python:<major>.<minor>-slim` images are based on Debian _stable_, the Python itself comes from some binary tarball, so only Debian base packages are in the  package database.  Base packages are mostly marked as `Essential`, and mostly lack dependencies.   They could disappear only on dist-upgrade, not on plain upgrade.

But I can add it if you feel strongly about it, or if you think we may use some more complicated base in future?

Reply:
But on the other, it does no harm either => added.

(And somebody could copy-pastes the update command to a container where it can help...)